docs/source/user-guide/general/metrics_guide.rst

```diff
@@ -0,0 +1,182 @@
+
+The default scorers by task type are:
+
+Tabular:
+
+Classification, binary or multiclass averaged over the classes:
+
+*   Accuracy
+*   Precision
+*   Recall
+
+Multiclass classification per class:
+
+*   F1
+*   Precision
+*   Recall
+
+Regression:
+
+*   Negative RMSE (negative so larger is better)
+*   Negative MAE
+*   R2
+
+Vision:
+
+Classification:
+
+*   Precision
+*   Recall
+
+Object detection:
+
+*   Mean average precision
+*   Mean average recall
+
+To run a check with the default metrics, run it without passing the scorers.
+
+:doc:`ClassPerformance </checks_gallery/vision/model_evaluation/plot_class_performance>`:
+
+.. code-block:: python
+
+    check = ClassPerformance()
+    result = check.run(train_ds, test_ds, mnist_model)
+
+
+Alternative metrics
+===================
+Sometimes the defaults aren’t good enough to describe the specifics of the problem.
+If this is the case, you can pass a list of metrics or a dict in the format {metric name: metric} as a parameter to the
+check.
+
+The metrics on the list can be existing
+`Ignite metrics <https://pytorch.org/ignite/metrics.html#complete-list-of-metrics>`__ or
+`Sklearn scorers <https://scikit-learn.org/stable/modules/model_evaluation.html>`__ or
+or a string from Deepchecks' `supported strings <#list-of-supported-strings>`__ or your own implementations.
+
+.. code-block:: python
+
+   train_ds, test_ds = adult.load_data(data_format='Dataset', as_train_test=True)
+   model = adult.load_fitted_model()
+
+   scorer = ['precision_per_class', 'recall_per_class', 'fnr_macro']
+   check = TrainTestPerformance(scorers=scorer)
+   result = check.run(train_ds, test_ds, model)
+   result
+
+
+List of Supported Strings
+=========================
+.. list-table:: Supported Strings
+   :widths: 25 75
+   :header-rows: 1
+
+   * - String
+     - Metric
+   * - 'neg_rmse'
+     - negative root mean squared error (Sklearn)
+   * - 'neg_mae'
+     - negative mean absolute error (Sklearn)
+   * - 'precision_per_class'
+     - precision per class - no averaging (results are returned per class)
+   * - 'recall_per_class'
+     - recall per class - no averaging
+   * - 'f1_per_class'
+     - F-1  per class - no averaging
+   * - 'fpr_per_class'
+     - False Positive Rate per class - no averaging
+   * - 'fpr'
+     - False Positive Rate - binary
+   * - 'fpr_macro'
+     - False Positive Rate - macro averaging
+   * - 'fpr_micro'
+     - False Positive Rate - micro averaging
+   * - 'fpr_weighted'
+     - False Positive Rate - weighted macro averaging
+   * - 'fnr_per_class'
+     - False Negative Rate per class - no averaging
+   * - 'fnr'
+     - False Negative Rate - binary
+   * - 'fnr_macro'
+     - False Negative Rate - macro averaging
+   * - 'fnr_micro'
+     - False Negative Rate - micro averaging
+   * - 'fnr_weighted'
+     - False Negative Rate - weighted macro averaging
+   * - 'tnr_per_class'
+     - True Negative Rate per class - no averaging
+   * - 'tnr'
```

Comment:
use specificity instead of tnr (maps to the same thing in backend)

Reply:
@Nadav-Barak I copies the list from the scorers module

Reply:
Can see at line 117 at scorers.py, we have mechanism to support in both specificity and sensitivity across all averaging methods